rdr_service/alembic/versions/5bb5c43c1b1f_add_fall_survey_to_ps_table.py

```diff
@@ -0,0 +1,62 @@
+"""add fall survey to ps table
+
+Revision ID: 5bb5c43c1b1f
+Revises: 18016ead2108, 058fde8817a7
+Create Date: 2021-08-17 14:59:41.047816
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = '5bb5c43c1b1f'
+down_revision = ('18016ead2108', '058fde8817a7')
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_vaccine_minute_2', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_vaccine_minute_2_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
```

Comment:
Did we stop adding the associated `questionnaire_on_*_time` fields to reflect when the (last) payload was received?  It looks like the summer COPE minute survey was the first one where we only added an `*_authored` field

Reply:
Yes, I think so.